pagopa/io-app
# feat: [IA-887] Disable the dark mode completely on Android

## Short description
This PR is going to disable completely the dark mode on Android. This will make all the icons like the "?" and the "X" visible again on a white background.

This PR fixes https://github.com/pagopa/io-app/issues/4008

## List of changes proposed in this pull request
- Changed the `AppTheme` to `Theme.AppCompat.Light.NoActionBar`

## How to test
Create an Android build and using both the dark mode and the light mode should not change how the app is displayed.
